packages/PoppyPrint-Tests.package/PPFormatterTest.class/instance/getConfigSpacesBeforePointinArray.st

```diff
@@ -0,0 +1,4 @@
+helper
+getConfigSpacesBeforePointinArray
```

Comment:
`Pointin` vs `PointIn`

Reply:
done

Reply:
> * one edge case you may want to add a test for is when an array is displayed on multiple lines

The edge case test for multiline arrays will be implemented in https://github.com/hpi-swa-teaching/poppy-print/issues/50